stats/lib/stats-fetchers.js

```diff
@@ -136,18 +136,15 @@ export const fetchParticipantChangeRates = async (pgPools, filter) => {
 
 /**
  * @param {PgPools} pgPools
- * @param {import('./typings.js').DateRangeFilter & {address: string}} filter
+ * @param {import('./typings.js').DateRangeFilter} filter
+ * @param {string} address
  */
-export const fetchParticipantScheduledRewards = async (pgPools, filter) => {
+export const fetchParticipantScheduledRewards = async (pgPools, { from, to }, address) => {
```

Comment:
As I mentioned in https://github.com/filecoin-station/spark-stats/pull/151, generic data (shared by all impact evaluators) that isn't specific to Spark should be exposed via routes & fetchers implemented in `stats/lib/platform-routes.js` and `stats/lib/platform-stats-fetcher.js`.

Could you please move "participant scheduled rewards" there?

Reply:
This isn't a platform feature, as it exposes scheduled rewards only from Spark. For Voyager it would come from voyager-stats, which would read from the voyager impact evaluator